editor/inspector/editor_inspector.cpp

```diff
@@ -3983,6 +3983,9 @@ void EditorInspector::update_tree() {
 		if ((p.usage & PROPERTY_USAGE_SCRIPT_VARIABLE) && name_style == EditorPropertyNameProcessor::STYLE_LOCALIZED) {
 			name_style = EditorPropertyNameProcessor::STYLE_CAPITALIZED;
 		}
+		if (path.begins_with("metadata/")) {
```

Comment:
We should add a comment above this `if` explaining why. Perhaps `// Metadata entries are accessed by String, so we should show the actual key name directly.`